packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -41,6 +42,9 @@ class PatchArtifactBundle {
   /// The artifact hash.
   final String hash;
 
+  /// The signature of the [hash].
+  final String? hashSignature;
```

Comment:
Minor but since `hash` and `hashSignature` are super closely related it might make sense to have a separate object for `Hash` like:

```dart
class Hash {
  final String value;
  final String? signature;
}
```

Reply:
I would call it `SignedHash` or `SignableHash` (or maybe even something like below), but this seems like a reasonable idea.

```dart
class ArtifactFingerprint {
  final String hash;
  final String? signature;
}
```

Reply:
We should also document that we expect this to be a base64-encoded string, or update the variable name to make this clear.

Reply:
Good idea, gonna refactor it